Create 1-3-E.cpp

johnhoth0 opened the pull request:
this submit: https://codeforces.com/group/R3IJoiTue4/contest/301990/submission/98171929

johnhoth0 committed:
Create 1-3-E.cpp

this submit: https://codeforces.com/group/R3IJoiTue4/contest/301990/submission/98171929

DmitryKorepanov commented on sources/1-3-E.cpp:
```diff
@@ -0,0 +1,119 @@
+	string s;
+	ll x, y;
+
+	map <int, set <ll>> sets;
```
нужно сделать класс, ввод-вывод оставить в main

johnhoth0 committed:
Update 1-3-E.cpp

moved to class
(https://codeforces.com/group/R3IJoiTue4/contest/301990/submission/101932111)

DmitryKorepanov commented on sources/1-3-E.cpp:
```diff
@@ -0,0 +1,136 @@
+	}
+	void listset(ll x) {
+		if (sets[x].empty()) {
+			cout << "-1\n";
```
ввод вывод только в Main
